std/process.d

```diff
@@ -3364,15 +3370,25 @@ static:
     Returns:
         the value of the environment variable if found, otherwise
         `null` if the environment doesn't exist.
-
-    Throws:
-    $(REF UTFException, std,utf) if the variable contains invalid UTF-16
-    characters (Windows only).
     */
-    string get(in char[] name, string defaultValue = null) @safe
+    string get(in char[] name, string defaultValue = null) @safe nothrow
     {
         string value;
-        auto found = getImpl(name, value);
+        bool found;
+        version(Windows)
+        {
+            try
+            {
+                found = getImplWindows(name, value);
+            }
+            catch (WindowsException) {}
+            catch (UTFException) {}
```

Comment:
This isn't right. We shouldn't be treating UTF-malformed variables or errors from the Windows API as if the variable doesn't exist. Silently ignoring errors in general is almost never a good idea.

Reply:
But having a function which is `nothrow` on Posix and throws on Windows is pretty ugly too.

Reply:
Well, you could throw an Error instead.

Reply:
Wait, it wasn't `nothrow` on posix, was it?

Reply:
Can we fix the documentation issue in a PR, and then work on the `nothrow` issue as a separate bugzilla enhancement/PR?

Reply:
I think throwing an `Error` would probably be fine. One could argue that this opens the window for a DoS attack by someone somehow sneaking bad UTF-16 into the environment (see Shellshock et al), but... this being Windows and all, I don't know if that's worth worrying about in this particular case.

Reply:
Also we probably want to throw Errors instead of UTFExceptions when auto-decoding bad UTF-8 anyway...

Reply:
Simplest and best solution, IMO: just don't make it `nothrow`. Why the heck does it need to be `nothrow` anyway? That the POSIX primitive never returns an error is really kind of a leaking implementation detail of how the environment is stored and encoded on whatever UNIX implementations POSIX was based off.